### PR TITLE
Release notes for v1.2.0

### PR DESCRIPTION
**Pull request by michael-cico**

### CODE REVIEW

File: content/en/docs/releasenotes/_index.md [hunk 1]
```diff
@@ -4,6 +4,42 @@ linkTitle: Release Notes
 weight: 13
 draft: false
 ---
+### v1.2.0
+Features:
+
+- Logging enhancements.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:10]: enhancements. -> enhancements: (colon)

File: content/en/docs/releasenotes/_index.md [hunk 2]
```diff
@@ -4,6 +4,42 @@ linkTitle: Release Notes
...
+  - Support for OCI Logging integration.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:11]: All the statements should be in past tense. 
Added support for OCI Logging integration.
Replaced
Updated
etc.

File: content/en/docs/releasenotes/_index.md [hunk 3]
```diff
@@ -4,6 +4,42 @@ linkTitle: Release Notes
...
+  - Replace Elasticsearch and Kibana with Opensearch and Opensearch Dashboard.
+  - Update Opensearch prod  profile data node configuration to 3 replicas.
+  - Enhanced Fluentd parsing/filtering rules for Verrazzano system logs.
+- Allow use of instance_principal auth with using OCI DNS.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:15]: All the statements should be in past tense.

File: content/en/docs/releasenotes/_index.md [hunk 4]
```diff
@@ -4,6 +4,42 @@ linkTitle: Release Notes
...
+- Support for metrics integration with non-OAM applications.
+- Add support for scaling Istio Gateways and setting Affinity.
+- Add support for scaling Verrazzano AuthProxy and setting Affinity.
+- Multiple image updates.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:19]: Updated multiple images (??? don't really know what that means)

File: content/en/docs/releasenotes/_index.md [hunk 5]
```diff
@@ -4,6 +4,42 @@ linkTitle: Release Notes
...
+- Component version updates.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:20]: updates. -> updates: (colon)

File: content/en/docs/releasenotes/_index.md [hunk 6]
```diff
@@ -4,6 +4,42 @@ linkTitle: Release Notes
...
+  
+Fixes:
+
+- Fixed Keycloak issue creating incorrect verrazzano-monitors group on install
+- Fixed Verrazzano failing to uninstall in a private registry configuration due to a missing Rancher image
+- Fixed Rancher install when tls-ca-additional secret is not present
+- Fixed Opensearch parsing errors of 'trait' field
+- Fixed Custom CA certs support
+- Fixed issue requeuing unsupported traits in the Verrazzano Application Operator, and update OAM Operator
+- Align Helidon workload service port names with Istio conventions to avoid protocol defaulting to TCP in all cases
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:37]: All the statements should be in past tense.
Aligned
Added

**Review by rosemarymarano — COMMENTED** [review]
Some things to fix.

**Review by JeffTancill — APPROVED** [review]

**Review by bryantidd — APPROVED** [review]

File: content/en/docs/releasenotes/_index.md (hunk 1 above)
Comment (michael-cico) [reply content/en/docs/releasenotes/_index.md:10]: ✅

File: content/en/docs/releasenotes/_index.md (hunk 2 above)
Comment (michael-cico) [reply content/en/docs/releasenotes/_index.md:11]: ✅

File: content/en/docs/releasenotes/_index.md (hunk 4 above)
Comment (michael-cico) [reply content/en/docs/releasenotes/_index.md:19]: I'll remove that.  I was going to say something else but was told not to.

File: content/en/docs/releasenotes/_index.md (hunk 5 above)
Comment (michael-cico) [reply content/en/docs/releasenotes/_index.md:20]: ✅

File: content/en/docs/releasenotes/_index.md (hunk 6 above)
Comment (michael-cico) [reply content/en/docs/releasenotes/_index.md:37]: ✅

File: content/en/docs/releasenotes/_index.md (hunk 4 above)
Comment (michael-cico) [reply content/en/docs/releasenotes/_index.md:19]: ✅

File: content/en/docs/releasenotes/_index.md (hunk 3 above)
Comment (michael-cico) [reply content/en/docs/releasenotes/_index.md:15]: ✅

File: content/en/docs/releasenotes/_index.md [hunk 7]
```diff
@@ -4,6 +4,41 @@ linkTitle: Release Notes
...
 ---
+### v1.2.0
+Features:
+
+- Logging enhancements:
+  - Added support for OCI Logging integration.
+  - Replaced Elasticsearch and Kibana with Opensearch and Opensearch Dashboard.
+  - Updated Opensearch `prod` profile data node configuration to 3 replicas.
+  - Enhanced Fluentd parsing/filtering rules for Verrazzano system logs.
+- Added support for using `instance_principal` auth with using OCI DNS.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:15]: auth with using -> authorization using

File: content/en/docs/releasenotes/_index.md [hunk 8]
```diff
@@ -4,6 +4,41 @@ linkTitle: Release Notes
...
+- Added support for metrics integration with non-OAM applications.
+- Added support for scaling Istio Gateways and setting Affinity.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:17]: Gateways -> gateways

File: content/en/docs/releasenotes/_index.md [hunk 9]
```diff
@@ -4,6 +4,41 @@ linkTitle: Release Notes
...
+  - MySQL v8.0.28.
+  - Grafana v7.5.11.
+  - Prometheus v2.31.1.
+  - Opensearch v1.2.3 (replaces Elasticsearch).
+  - Opensearch Dashboards v1.2.0 (replaces Kibana).
+  - WebLogic Kubernetes Operator v3.3.7.
+  
+Fixes:
+
+- Fixed Keycloak issue creating incorrect verrazzano-monitors group on install
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:30]: verrazzano-monitors group on install -> `verrazzano-monitors` (code font) group on installation. (period at the end of each sentence; please fix globally)

File: content/en/docs/releasenotes/_index.md [hunk 10]
```diff
@@ -4,6 +4,41 @@ linkTitle: Release Notes
...
+- Fixed Verrazzano failing to uninstall in a private registry configuration due to a missing Rancher image
+- Fixed Rancher install when tls-ca-additional secret is not present
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:32]: install when tls-ca-additional secret -> installation when `tls-ca-additional` (code font) secret

File: content/en/docs/releasenotes/_index.md [hunk 11]
```diff
@@ -4,6 +4,41 @@ linkTitle: Release Notes
...
+- Fixed Opensearch parsing errors of `trait` field
+- Fixed Custom CA certs support
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:34]: certs  -> certificates

File: content/en/docs/releasenotes/_index.md [hunk 12]
```diff
@@ -4,6 +4,41 @@ linkTitle: Release Notes
...
+- Fixed issue requeuing unsupported traits in the Verrazzano Application Operator, and update OAM Operator
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:35]: and update OAM Operator -> and updated OAM Operator

**Review by rosemarymarano — COMMENTED** [review]
Round Two edits.

File: content/en/docs/releasenotes/_index.md (hunk 12 above)
Comment (michael-cico) [reply content/en/docs/releasenotes/_index.md:35]: ✅

File: content/en/docs/releasenotes/_index.md (hunk 9 above)
Comment (michael-cico) [reply content/en/docs/releasenotes/_index.md:30]: ✅

File: content/en/docs/releasenotes/_index.md (hunk 8 above)
Comment (michael-cico) [reply content/en/docs/releasenotes/_index.md:17]: ✅

File: content/en/docs/releasenotes/_index.md (hunk 7 above)
Comment (michael-cico) [reply content/en/docs/releasenotes/_index.md:15]: ✅

File: content/en/docs/releasenotes/_index.md (hunk 8 above)
Comment (michael-cico) [reply content/en/docs/releasenotes/_index.md:17]: also changed `Affinity` --> `affinity`

**Review by rosemarymarano — APPROVED** [review]
Perfect. Thank you.

**Pull request merged**